Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -1,12 +1,31 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.EventSystems;
 
 namespace HoloToolkit.Unity.InputModule
 {
+    [Serializable]
+    public class PointerResult
```

Comment:
Should be in it's own class unless it can be made private

Reply:
Why can't we add these new properties to the PointerData class?

Reply:
I'll break it out.

Also: I could use everyone's input on this class.

IPointingSource needs to be able to access the result of its raycasts for drawing purposes. Eg the parabola needs to be able to see where it 'stopped' and only draw that far.

But because FocusManager.PointerData is a private class, we can't pass that directly back to the pointer.

The path of least resistance was to make the private class FocusManager.PointerData extend a new *public* class, which is PointerResult. PointerResult contains only the information it outside classes need to know, while FocusManager.PointerData impelements what the FocusManager needs to know.

This works but it's hella ugly and there's got to be a better way. Open to suggestions.

Reply:
I think it makes sense to make PointerData public and protect the setters.